packages/docs/content/5_components/badge.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Badge
+relatedUswdsGuidance: components/tag/
+---
+
+Badges hold small amounts of information and draw attention to new or important content.
+
+## `<Badge>`
+
+<StorybookExample
```

Comment:
I found by adding the prettier option `htmlWhitespaceSensitivity: "ignore"` 

```javascript
  const prettyHtml = prettier.format(html, { htmlWhitespaceSensitivity: 'ignore', parser: 'html', plugins: [parserHtml] });
```

The code formatting looks a lot better for me, on badge particularly it was wrapping in a strange way without this. Maybe can set that globally?